.travis.yml

```diff
@@ -17,10 +17,11 @@ before_install:
     - ./scripts/install-dependencies.rb
 
 script:
-    - ./scripts/install-robolectric.sh
+    - ./gradlew clean assemble publishToMavenLocal compileTest --info --stacktrace
```

Comment:
At the moment, yes, the classloader is looking in the maven local repos for shadows. I'd like to make gradle provide a list of paths to search (`shadows-core/build/classes` etc) but I think that's best left to a subsequent change.